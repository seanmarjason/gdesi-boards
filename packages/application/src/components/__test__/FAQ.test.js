import { render } from '@testing-library/react'
import FAQ from '../FAQ';

it('FAQ renders correctly', () => {
  const { container } = render(<FAQ />)
  expect(container).toMatchSnapshot()
})
