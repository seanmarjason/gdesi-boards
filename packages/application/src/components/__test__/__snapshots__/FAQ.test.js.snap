// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAQ renders correctly 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-hh8zal-MuiContainer-root"
    id="faq"
  >
    <h2
      class="MuiTypography-root MuiTypography-h4 css-1hviwvq-MuiTypography-root"
    >
      Frequently asked questions
    </h2>
    <div
      class="MuiBox-root css-8atqhb"
    >
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded MuiAccordion-gutters css-1808mag-MuiPaper-root-MuiAccordion-root"
        style="--Paper-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);"
      >
        <h3
          class="MuiAccordion-heading css-cy7rkm-MuiAccordion-heading"
        >
          <button
            aria-controls="panel1d-content"
            aria-expanded="false"
            class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-1f1pi2l-MuiButtonBase-root-MuiAccordionSummary-root"
            id="panel1d-header"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiAccordionSummary-content MuiAccordionSummary-contentGutters css-yfrx4k-MuiAccordionSummary-content"
            >
              <span
                class="MuiTypography-root MuiTypography-subtitle2 css-1ypmsg6-MuiTypography-root"
              >
                How do I contact customer support if I have a question or issue?
              </span>
            </span>
            <span
              class="MuiAccordionSummary-expandIconWrapper css-1wqf3nl-MuiAccordionSummary-expandIconWrapper"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
                data-testid="ExpandMoreIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
                />
              </svg>
            </span>
          </button>
        </h3>
        <div
          class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-cwrbtg-MuiCollapse-root"
          style="min-height: 0px;"
        >
          <div
            class="MuiCollapse-wrapper MuiCollapse-vertical css-1x6hinx-MuiCollapse-wrapper"
          >
            <div
              class="MuiCollapse-wrapperInner MuiCollapse-vertical css-1i4ywhz-MuiCollapse-wrapperInner"
            >
              <div
                aria-labelledby="panel1d-header"
                class="MuiAccordion-region"
                id="panel1d-content"
                role="region"
              >
                <div
                  class="MuiAccordionDetails-root css-1lj39kh-MuiAccordionDetails-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body2 MuiTypography-gutterBottom css-g3k0xs-MuiTypography-root"
                  >
                    You can reach our customer support team by emailing 
                    <a
                      class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1orsvah-MuiTypography-root-MuiLink-root"
                      href="mailto:support@email.com"
                    >
                      support@email.com
                    </a>
                     or calling our toll-free number. We're here to assist you promptly.
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded MuiAccordion-gutters css-1808mag-MuiPaper-root-MuiAccordion-root"
        style="--Paper-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);"
      >
        <h3
          class="MuiAccordion-heading css-cy7rkm-MuiAccordion-heading"
        >
          <button
            aria-controls="panel2d-content"
            aria-expanded="false"
            class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-1f1pi2l-MuiButtonBase-root-MuiAccordionSummary-root"
            id="panel2d-header"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiAccordionSummary-content MuiAccordionSummary-contentGutters css-yfrx4k-MuiAccordionSummary-content"
            >
              <span
                class="MuiTypography-root MuiTypography-subtitle2 css-1ypmsg6-MuiTypography-root"
              >
                Can I return the product if it doesn't meet my expectations?
              </span>
            </span>
            <span
              class="MuiAccordionSummary-expandIconWrapper css-1wqf3nl-MuiAccordionSummary-expandIconWrapper"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
                data-testid="ExpandMoreIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
                />
              </svg>
            </span>
          </button>
        </h3>
        <div
          class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-cwrbtg-MuiCollapse-root"
          style="min-height: 0px;"
        >
          <div
            class="MuiCollapse-wrapper MuiCollapse-vertical css-1x6hinx-MuiCollapse-wrapper"
          >
            <div
              class="MuiCollapse-wrapperInner MuiCollapse-vertical css-1i4ywhz-MuiCollapse-wrapperInner"
            >
              <div
                aria-labelledby="panel2d-header"
                class="MuiAccordion-region"
                id="panel2d-content"
                role="region"
              >
                <div
                  class="MuiAccordionDetails-root css-1lj39kh-MuiAccordionDetails-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body2 MuiTypography-gutterBottom css-g3k0xs-MuiTypography-root"
                  >
                    Absolutely! We offer a hassle-free return policy. If you're not completely satisfied, you can return the product within [number of days] days for a full refund or exchange.
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded MuiAccordion-gutters css-1808mag-MuiPaper-root-MuiAccordion-root"
        style="--Paper-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);"
      >
        <h3
          class="MuiAccordion-heading css-cy7rkm-MuiAccordion-heading"
        >
          <button
            aria-controls="panel3d-content"
            aria-expanded="false"
            class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-1f1pi2l-MuiButtonBase-root-MuiAccordionSummary-root"
            id="panel3d-header"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiAccordionSummary-content MuiAccordionSummary-contentGutters css-yfrx4k-MuiAccordionSummary-content"
            >
              <span
                class="MuiTypography-root MuiTypography-subtitle2 css-1ypmsg6-MuiTypography-root"
              >
                What makes your product stand out from others in the market?
              </span>
            </span>
            <span
              class="MuiAccordionSummary-expandIconWrapper css-1wqf3nl-MuiAccordionSummary-expandIconWrapper"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
                data-testid="ExpandMoreIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
                />
              </svg>
            </span>
          </button>
        </h3>
        <div
          class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-cwrbtg-MuiCollapse-root"
          style="min-height: 0px;"
        >
          <div
            class="MuiCollapse-wrapper MuiCollapse-vertical css-1x6hinx-MuiCollapse-wrapper"
          >
            <div
              class="MuiCollapse-wrapperInner MuiCollapse-vertical css-1i4ywhz-MuiCollapse-wrapperInner"
            >
              <div
                aria-labelledby="panel3d-header"
                class="MuiAccordion-region"
                id="panel3d-content"
                role="region"
              >
                <div
                  class="MuiAccordionDetails-root css-1lj39kh-MuiAccordionDetails-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body2 MuiTypography-gutterBottom css-g3k0xs-MuiTypography-root"
                  >
                    Our product distinguishes itself through its adaptability, durability, and innovative features. We prioritize user satisfaction and continually strive to exceed expectations in every aspect.
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiAccordion-root MuiAccordion-rounded MuiAccordion-gutters css-1808mag-MuiPaper-root-MuiAccordion-root"
        style="--Paper-shadow: 0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);"
      >
        <h3
          class="MuiAccordion-heading css-cy7rkm-MuiAccordion-heading"
        >
          <button
            aria-controls="panel4d-content"
            aria-expanded="false"
            class="MuiButtonBase-root MuiAccordionSummary-root MuiAccordionSummary-gutters css-1f1pi2l-MuiButtonBase-root-MuiAccordionSummary-root"
            id="panel4d-header"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiAccordionSummary-content MuiAccordionSummary-contentGutters css-yfrx4k-MuiAccordionSummary-content"
            >
              <span
                class="MuiTypography-root MuiTypography-subtitle2 css-1ypmsg6-MuiTypography-root"
              >
                Is there a warranty on the product, and what does it cover?
              </span>
            </span>
            <span
              class="MuiAccordionSummary-expandIconWrapper css-1wqf3nl-MuiAccordionSummary-expandIconWrapper"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1umw9bq-MuiSvgIcon-root"
                data-testid="ExpandMoreIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M16.59 8.59 12 13.17 7.41 8.59 6 10l6 6 6-6z"
                />
              </svg>
            </span>
          </button>
        </h3>
        <div
          class="MuiCollapse-root MuiCollapse-vertical MuiCollapse-hidden css-cwrbtg-MuiCollapse-root"
          style="min-height: 0px;"
        >
          <div
            class="MuiCollapse-wrapper MuiCollapse-vertical css-1x6hinx-MuiCollapse-wrapper"
          >
            <div
              class="MuiCollapse-wrapperInner MuiCollapse-vertical css-1i4ywhz-MuiCollapse-wrapperInner"
            >
              <div
                aria-labelledby="panel4d-header"
                class="MuiAccordion-region"
                id="panel4d-content"
                role="region"
              >
                <div
                  class="MuiAccordionDetails-root css-1lj39kh-MuiAccordionDetails-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body2 MuiTypography-gutterBottom css-g3k0xs-MuiTypography-root"
                  >
                    Yes, our product comes with a [length of warranty] warranty. It covers defects in materials and workmanship. If you encounter any issues covered by the warranty, please contact our customer support for assistance.
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
